# HG changeset patch
# User Felix Janda <felix.janda@posteo.de>
# Date 1429127179 -7200
#      Wed Apr 15 21:46:19 2015 +0200
# Node ID 1add4a9525c1c51521927203e7fe916ef7bbfe6a
# Parent  53ceefb0e1c80ded0f3c154e26b96b48c8821210
profiler: avoid getcontext and gettid

diff -r 53ceefb0e1c8 -r 1add4a9525c1 tools/profiler/platform-linux-lul.h
--- a/tools/profiler/platform-linux-lul.h	Wed Apr 15 15:33:19 2015 -0400
+++ b/tools/profiler/platform-linux-lul.h	Wed Apr 15 21:46:19 2015 +0200
@@ -18,7 +18,7 @@
 
 // We need a definition of gettid(), but glibc doesn't provide a
 // wrapper for it.
-#if defined(__GLIBC__)
+#if 1
 #include <unistd.h>
 #include <sys/syscall.h>
 static inline pid_t gettid()
diff -r 53ceefb0e1c8 -r 1add4a9525c1 tools/profiler/platform-linux.cc
--- a/tools/profiler/platform-linux.cc	Wed Apr 15 15:33:19 2015 -0400
+++ b/tools/profiler/platform-linux.cc	Wed Apr 15 21:46:19 2015 +0200
@@ -701,10 +701,10 @@
 {
   MOZ_ASSERT(aContext);
   ucontext_t* pContext = reinterpret_cast<ucontext_t*>(aContext);
-  if (!getcontext(pContext)) {
+/*  if (!getcontext(pContext)) {
     context = pContext;
     SetSampleContext(this, aContext);
-  }
+  } */
 }
 
 void OS::SleepMicro(int microseconds)
